extensions/ql-vscode/src/common/logging/tee-logger.ts

```diff
@@ -50,6 +58,7 @@ export class TeeLogger implements Logger {
       } catch (e) {
         // Write an error message to the primary log, and stop trying to write to the side log.
         this.error = true;
+        this.fileHandle = undefined;
```

Comment:
Are you sure we shouldn't call `await this.fileHandle?.close()` here? I'm wondering if the exception could have been from something that didn't come from the file not being open correctly. In that case we'll leak the file handle and open a new one next time we try to log.

Reply:
Good point, we should close it here. There isn't a risk that we open a new file handle to the same file next time we try to log since `this.error` is never reset, but we won't close the file handle in `dispose` either (since `this.fileHandle` is now `undefined`), so we should close it here.